backend/api/tests/test_venueapi.py

```diff
@@ -0,0 +1,548 @@
+"""Tests for api/venueapi.py."""
+
+from api.venueapi import VenueApi
+
+
+def test_get_details():
+    """Tests get_details()."""
+    test_venue = VenueApi("4c55b289f5f3d13ad7f539fc")
+
+    result = test_venue.get_details(succesfull_venue_response)
+
+    assert isinstance(result["name"], str)
+    assert isinstance(result["address"], list)
+    assert isinstance(result["latitude"], float)
+    assert isinstance(result["longitude"], float)
+    assert isinstance(result["categories"], list)
+    assert isinstance(result["photo"], str)
+    assert isinstance(result["attributes"], list)
```

Comment:
Could we just check the actual result?
```suggestion
    assert result == {...}
```